exercises/rectangles/tests/rectangles.rs

```diff
@@ -36,64 +36,42 @@ fn test_one_rectangle() {
 #[test]
 #[ignore]
 fn test_two_rectangles_no_shared_parts() {
-    let lines = vec![
-        "  +-+",
-        "  | |",
-        "+-+-+",
-        "| |  ",
-        "+-+  "
-        ];
+    let lines = vec!["  +-+", "  | |", "+-+-+", "| |  ", "+-+  "];
```

Comment:
Should be able to add `#[cfg_attr(rustfmt, rustfmt_skip)]` to the test, see [rustfmt tips](https://github.com/rust-lang-nursery/rustfmt#tips).